### ISSUE TITLE
time estimate for stable release

### ISSUE THREAD

**Issue by qussaif10** [minio/minio-rs#98]
i saw that the rust sdk is not listed on the website, so i'm assuming it is not stable yet to use, can anyone give me an estimate of when it might be stable enough to use?

**Comment by HJLebbink** [comment]
Yes, it's not listed. I'll have a quick word with some and see what the consensus is.

**Comment by vnghia** [comment]
Hi @HJLebbink, do you have any timeline in mind yet ? Thank you!
